### PR TITLE
Modify GSoC part and add news about meeting 2019

### PR DESCRIPTION
**Pull request by MihirLuthra**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -37,8 +37,24 @@
...
+        </div>
+        <div class="column">
+          <a href="<?php print $trac_url . 'wiki/SummerOfCode'; ?>"> <img src="img/gsoc.png"> </a>
+        </div>
+    </div>
+
+<!-- Comment GSoC code for now
     <div class="row">
         <div class="column">
             <p> <b>Google Summer of Code</b> is a program to encourage students' participation in Open Source development and offers a stipend to work on the project with an organization for three months. MacPorts has been participating in the program since 2007! </p>
```
Comment (raimue) [inline index.php]: I like the idea about linking directly to past GSoC projects. However, this first paragraph is a good description of what "GSoC" actually is. I would leave that in, otherwise there is no explanation of what "GSoC" refers to.

Also, please do not remove the link to the ideas page. Providing more information to interested students is relevant all year.

Comment (MihirLuthra) [reply index.php]: @raimue, I made the changes. Also, I added horizontal rows in between shortcuts. I feel that makes it more readable. I will withdraw that commit if it looks inappropriate.

**Comment by mojca** [comment]
I would usually provide more feedback and let you fix stuff, but I made a few minor corrections now since I wanted to deploy the changes fast. Thank you very much for the contribution and for writing the news piece.

I'll leave the commit to `headers.inc` for others to discuss. (Do you happen to have a screenshot?)

**Comment by mojca** [comment]
I'm sorry for stealing your PR for this. This is what I would do with the GSOC logo (if that's the correct approach):

image

```html
    <div class="row">
        <a href="<?php print $trac_url . 'wiki/SummerOfCode'; ?>"> <img src="img/gsoc.png" width="110" height="110" style="float: right"> </a>
        <p>A good way for students to get involved is through the <a href="https://summerofcode.withgoogle.com">Google Summer of Code</a>. GSoC is a program to encourage students' participation in Open Source development and offers a stipend to work on the project with an organization for three months. MacPorts has been participating in the program since 2007! We shall participate next year as well. You may find <a href="<?php print $trac_url . 'wiki/SummerOfCodeArchive'; ?>">past GSoC projects here</a>.</p>

<!-- Comment GSoC participation for now
            <p>We are looking for students to join us in making <?php print date('Y'); ?> the best Summer of Code yet. Applicants will be paired with The MacPorts Project mentors to guide them through the coding tasks. This is a great opportunity to gain exposure to real-world software development scenarios. You don't have to be a proven developer, we'd like to have students who are committed to helping MacPorts. </p>
-->
    </div>

    <div class="row">
        <p> We have a list of ideas with possible tasks for MacPorts and additional information about the process at <a href="<?php print $trac_url . 'wiki/SummerOfCode'; ?>">wiki/SummerOfCode</a>. We are always open to new ideas. Research on the idea, draft an initial proposal and get it reviewed. </p>
    </div>
```

**Comment by MihirLuthra** [comment]
> I would usually provide more feedback and let you fix stuff, but I made a few minor corrections now since I wanted to deploy the changes fast. Thank you very much for the contribution and for writing the news piece.
> 
> I'll leave the commit to `headers.inc` for others to discuss. (Do you happen to have a screenshot?)
Screenshot 2019-09-06 at 7 33 07 PM

I thought the shortcuts menu is a bit cluttered due to multiline content, so I just inserted `<hr>` tags in between. Although there might be better ways to do the same but I am am new to html, css and stuff so did the best upto my knowledge.

**Comment by MihirLuthra** [comment]
> I'm sorry for stealing your PR for this. This is what I would do with the GSOC logo (if that's the correct approach):
> 
> image
> 
> ```
>     <div class="row">
>         <a href="<?php print $trac_url . 'wiki/SummerOfCode'; ?>">  </a>
>         <p>A good way for students to get involved is through the <a href="https://summerofcode.withgoogle.com">Google Summer of Code</a>. GSoC is a program to encourage students' participation in Open Source development and offers a stipend to work on the project with an organization for three months. MacPorts has been participating in the program since 2007! We shall participate next year as well. You may find <a href="<?php print $trac_url . 'wiki/SummerOfCodeArchive'; ?>">past GSoC projects here</a>.</p>
> 
> 
>     </div>
> 
>     <div class="row">
>         <p> We have a list of ideas with possible tasks for MacPorts and additional information about the process at <a href="<?php print $trac_url . 'wiki/SummerOfCode'; ?>">wiki/SummerOfCode</a>. We are always open to new ideas. Research on the idea, draft an initial proposal and get it reviewed. </p>
>     </div>
> ```

No worries at all, it looks better undoubtely.

File: includes/header.inc [hunk 2]
```diff
@@ -92,11 +92,16 @@
                             'MacPorts Team' => $trac_url . 'wiki/MacPortsDevelopers',
                             'Becoming a Member' => $guide_url . '#project.membership',
                         );
-                        foreach ($shortcuts as $title => $url) {
-                            if (basename($_SERVER['PHP_SELF']) == $url) {
+						foreach ($shortcuts as $title => $url) {
+							if (basename($_SERVER['PHP_SELF']) == $url) {
```
Comment (raimue) [inline includes/header.inc]: These whitespace changes seem unrelated and unnecessary. The rest of the file uses spaces, not tabs. New code should follow the existing style.

Comment (MihirLuthra) [reply includes/header.inc]: @raimue done. Sorry I always forget to change `.vimrc` back to use tabs. Will stick to the new changes.

**Comment by raimue** [comment]
I am a bit confused whether this last rebase/force-push lost some changes or what happened. I only see the changes regarding the `<hr />` lines now. I think @mojca already pushed the rest to master?

**Comment by mojca** [comment]
> I am a bit confused whether this last rebase/force-push lost some changes or what happened. I only see the changes regarding the `<hr>` lines now. I think @mojca already pushed the rest to master?

Yes, I pushed the changes for which this PR was mainly for already to the master branch, as I wanted to get the news out as fast as possible. I wanted to get input/feedback about design from someone else.
